setup.py

```diff
@@ -138,8 +138,7 @@ def get_install_requirements(install_requires, choose_install_requires):
     long_description_content_type="text/markdown",
     packages=find_packages(
         exclude=["app", "eta", "package", "requirements", "tests", "tools"]
-    )
-    + ["fiftyone.recipes", "fiftyone.tutorials"],
+    ),
     package_dir={
```

Comment:
Awesome! I believe we can remove `package_dir` kwarg, as well

Reply:
Done!